CYCLONEDDS_QUALITY_DECLARATION.md

```diff
@@ -14,65 +14,90 @@ Below are the rationales, notes, and caveats for this claim, organized by each r
+`CycloneDDS` will not break public API within a released ROS distribution, i.e. no major releases once the ROS distribution is released. It conforms to the [Releases section](https://www.eclipse.org/projects/handbook/#release) of the Eclipse project handbook which states that releases which include breaking API changes are considered "Major".
 
 ### ABI Stability Policy [1.v]
 
-There is no published policy for ABI stability in `CycloneDDS`.
+`CycloneDDS` provides ABI stability for PATCH release. `CycloneDDS` strives to provide ABI stability for MINOR release. `CycloneDDS` not guarantee ABI stability for MAJOR release.
 
 ### ABI and ABI Stability Within a Released ROS Distribution [1.vi]
 
+`CycloneDDS` will not break API nor ABI within a released ROS distribution, i.e. no major releases once the ROS distribution is released. 
 As an external package, `CycloneDDS` is not released on the same cadence as ROS.
 Changes to the branch of the `CycloneDDS` repository that is targeted by a given ROS release would be picked up by development builds of that ROS release.
-There is no published policy for what criteria are required for changes that are accepted in those branches.
 
-## Change Control Process [2]
+## Change Control [2]
+
+`CycloneDDS` follows the recommended guidelines of the [Eclipse Development Process](https://www.eclipse.org/projects/dev_process/).
+
+For the RMW layer, we follow the ROS core packages process. For Eclipse Cyclone DDS, we ensure the stability within the project through review, CI and tests and additionally run ROS CI for changes that are likely to affect ROS.
+
+All commits must be signed by the author and there must be an [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php) on file with the Eclipse Foundation.
+
+Pull requests are required to pass all tests in the CI system, unless Committers consider there is sufficient evidence that a failure is the result of a mishap unrelated to the change. Pull requests are only merged if the Committers deem it of acceptable quality and provide sufficient coverage of new functionality or proof of a bug fix via tests.
+
+Only the Committers have write access to the repository and they are voted for in elections monitored by the Eclipse Foundation staff.
 
 ### Change Requests [2.i]
 
-The source code repository for `CycloneDDS` is hosted in GitHub, and the change process follows a typical GitHub pull request model.
-There are several [team members](https://projects.eclipse.org/projects/iot.cyclonedds/who) with rights to merge the pull requests, and an election process exists for promoting new members of that team.
+All changes will occur through a pull request, check ROS 2 Developer Guide for additional information.
```

Comment:
Not the ROS 2 developer guid but the process described in the [Eclipse Handbook](https://www.eclipse.org/projects/handbook/#contributing-contributors)